asm/macros/battle_script.inc

```diff
@@ -1669,6 +1669,12 @@
 	callnative BS_DamageToQuarterTargetHP
 	.endm
 
+	.macro jumpifsleepclause battler:req, jumpInstr:req
+	callnative BS_JumpIfSleepClause
+	.byte \battler
```

Comment:
you don't need the battle argument here

Reply:
isnt it actually used in the function?

Reply:
`jumpifsleepclause` uses `BS_TARGET` as the argument which is `gBattlerTarget`but `BS_TARGET` needs to be first "transformed" into a battler argument. 

If you check the callnative you already checked gBattlerTarget for the majority of them